cluster/manifests/skipper/deployment.yaml

```diff
@@ -45,9 +45,9 @@ spec:
             matchLabels:
               application: skipper-ingress
 {{- end }}
-      priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
+      priorityClassName: "{{ .ConfigItems.system_priority_class }}"
       serviceAccountName: skipper-ingress
-      dnsPolicy: ClusterFirstWithHostNet
```

Comment:
I think that's not supposed to be this way, probably the automation went too far?